README.md

```diff
@@ -267,29 +267,29 @@ compdef kubecolor=kubectl
 
 Fish completion is officially unsupported by `kubectl`, so it is unsupported by `kubecolor` as well.
 
-However, there are 2 ways we can make them work. Please keep in mind these are a kind of "hack" and not officially supported.
+However, there is a way we can make them work. Please keep in mind these are a kind of "hack" and not officially supported.
 
-1. Use [evanlucas/fish-kubectl-completions](https://github.com/evanlucas/fish-kubectl-completions) with `kubecolor`:
+1. Install `kubectl` completions (https://github.com/evanlucas/fish-kubectl-completions):
```

Comment:
This project is a bit old by now, and according to #87 we possibly don't even need any extensions for fish support.

I haven't gotten to tackle #87, but maybe since you're at it, could you look into if we could skip the `fisher install` step completely?

Reply:
I was not aware of this, this is indeed a preferred solution and is working great!